packages/textInput/components/utils.tsx

```diff
@@ -78,7 +78,7 @@ const getInputElement = (
 };
 
 const getInputAppearance = (props: TextInputProps): string => {
-  return props.disabled ? "disabled" : props.appearance;
```

Comment:
Is there a better solution for this? WIthout the `!`, I get the following TypeScript warning:
```
Type 'string | undefined' is not assignable to type 'string'.
```

Reply:
Great question! We generally try to avoid using `!` in our codebase.

There are [several methods](https://bobbyhadz.com/blog/typescript-argument-type-undefined-not-assignable-parameter-type-string) to fix this. 
I might suggest using some nullish coalescing here, like this:
```
const getInputAppearance = (props: TextInputProps): string => {
  return props.disabled
    ? "disabled"
    : props.appearance ?? InputAppearance.Standard;
};
```

If appearance is undefined or null, explicitly set the fallback to the default defined in `TextInputProps`.
```
  /**
   * Sets the current appearance of the input component. This defaults to InputAppearance.Standard, but supports `InputAppearance.Error` & `InputAppearance.Success` appearances as well.
   */
  appearance?: InputAppearance;
   ```
   
It might be even more readable to use an if statement in there instead. But I'll leave that up to you!

```
const getInputAppearance = (props: TextInputProps): string => 
{
  if (props.disabled) {
    return "disabled";
  }
  return props.appearance ?? InputAppearance.Standard;
};
```

I hope that helps as another option here!

Reply:
Hey @nataliepina, thanks so much for this explanation! This was super helpful for me 🙌